mysql-test/suite/rocksdb/t/rocksdb_datadir.test

```diff
@@ -1,8 +1,13 @@
 --source include/have_rocksdb.inc
 
+if (!$ROCKSDB_LOAD_PATH) {
```

Comment:
That I am not sure of which is why I added the extra checking. My initial assumption was that no, if we pass have_rocksdb, then all of these plugin.defs should be defined, except that if the plugin.def for rocksdb were to change in a way that allowed some of the env vars to be defined but not all. I can't imagine such a scenario off the top of my head, but it may be possible.